src/builder/arg.rs

```diff
@@ -3880,6 +3880,13 @@ impl Arg {
         })
     }
 
+    /// Get the value terminator for this argument. The value_terminator is a value
+    /// that terminates parsing of multi-valued arguments.
+    #[inline]
+    pub fn get_value_terminator(&self) -> Option<&Str> {
```

Comment:
nit: can you put this just after `get_value_delimiter`?  Thats the way the setters are ordered